script/update_outcomes.csv

```diff
@@ -7,7 +7,7 @@ Israel,2,/world/organisations/british-embassy-tel-aviv,true
 Netherlands,2,/world/organisations/british-embassy-the-hague,true
 USA,2,/world/organisations/british-embassy-washington,true
 Bermuda,3,FALSE,true
-the Falkland Islands,3,FALSE,
+the Falkland Islands,3,FALSE,true
```

Comment:
Minor: it's a bit weird that we use uppercase `FALSE` if there is no URL to point to, we use "" if a country hasn't been processed, and we use lowercase `true` when it has 😅 
But changing that would be a lot of work - so I'm happy to ignore!